ui-frontend/packages/catalog-ui-search/src/main/webapp/component/tabs/metacard/summary.tsx

```diff
@@ -432,153 +435,179 @@ const AttributeComponent = ({
-                                  style={{
-                                    maxWidth: '100%',
-                                    maxHeight: '50vh',
-                                  }}
-                                />
-                              </a>
-                            )
-                          case 'BOOLEAN':
-                            return (
-                              <Typography>{val ? 'true' : 'false'}</Typography>
-                            )
-                          default:
-                            if (lazyResult.highlights[attr]) {
-                              if (attr === 'title') {
-                                //Special case, title highlights don't get truncated
+                            case 'BINARY':
+                              return (
+                                <a
+                                  target="_blank"
+                                  href={TypedMetacardDefs.getImageSrc({ val })}
+                                  style={{ padding: '0px' }}
+                                >
+                                  <img
+                                    src={TypedMetacardDefs.getImageSrc({ val })}
+                                    style={{
+                                      maxWidth: '100%',
+                                      maxHeight: '50vh',
+                                    }}
+                                  />
+                                </a>
+                              )
+                            case 'BOOLEAN':
+                              return (
+                                <Typography>
+                                  {val ? 'true' : 'false'}
+                                </Typography>
+                              )
+                            default:
+                              if (lazyResult.highlights[attr]) {
+                                if (attr === 'title') {
+                                  //Special case, title highlights don't get truncated
+                                  return (
+                                    <Typography>
+                                      <span
+                                        dangerouslySetInnerHTML={{
+                                          __html:
+                                            lazyResult.highlights[attr][0]
+                                              .highlight,
+                                        }}
+                                      />
+                                    </Typography>
+                                  )
+                                }
+                                return displayHighlightedAttrInFull(
+                                  lazyResult.highlights[attr],
+                                  val,
+                                  index
+                                )
+                              } else if (isUrl(val)) {
```

Comment:
❓ I think this is fine as is, but should we do something special if it is a link and also highlighted? I'll defer to @lifenouveau here.

Reply:
Theoretically the background would change, right?

Reply:
Taking a look at this logic again, it only shows it as a url if it isn't highlighted. 🤔

Reply:
I see, ideally we have both.

Reply:
Ahhh I didn't realize highlighted meant exactly what you'd think it means. Will refactor so it links it for highlighted attributes as well.

Reply:
There's an issue where the <a> tag overwrites the inner highlight formatting, so I'm highlighting the entire link